deepspeed/runtime/fp16/fused_optimizer.py

```diff
@@ -181,7 +181,7 @@ def step_fused_adam(self, closure=None):
                                                      apply_scale=False)
 
         # Stash unscaled gradient norm
-        self._global_grad_norm = scaled_global_grad_norm / self.cur_scale
+        self._global_grad_norm = scaled_grad_norm / self.cur_scale
```

Comment:
Is this to fix a bug? why was this changed?

Reply:
`scaled_global_grad_norm` was undefined. However, there is a `scaled_grad_norm = get_global_norm(norm_list=norm_groups)` a couple of lines up, so I assumed that's what was intended instead.